client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -179,6 +179,12 @@ class SiteSettingsFormDiscussion extends Component {
 					onChange={ this.handleCommentOrder }>
 					<span>{ translate( 'Comments should be displayed with the older comments at the top of each page' ) }</span>
 				</CompactFormToggle>
+				<CompactFormToggle
```

Comment:
This setting is intended to activate/deactivate a module and not a module setting. For module toggles we usually use a `<JetpackModuleToggle />`. It will probably be a little shorter to use it instead of the `<CompactFormToggle />`, and it's a consistent pattern we've been following for module activation/deactivation toggles.